test/run_test_bort.sh

```diff
@@ -9,171 +9,178 @@
-# is resolved.
-# Check string().
-#./test_bort_4 string 1
-#[ $? != 1 ] &&  exit 1
-
-# Check ufbcnt().
-./test_bort_4 ufbcnt 1
-[ $? != 1 ] &&  exit 1
-
-# Check ufbcpy().
-./test_bort_4 ufbcpy 1
-[ $? != 1 ] &&  exit 1
-
-# Check ufbcup().
-./test_bort_4 ufbcup 1
-[ $? != 1 ] &&  exit 1
-
-# Check ufbdmp().
-./test_bort_4 ufbdmp 1
-[ $? != 1 ] &&  exit 1
-
-# Check ufbevn().
-./test_bort_4 ufbevn 1
-[ $? != 1 ] &&  exit 1
-
-# Check ufbget().
-./test_bort_4 ufbget 1
-[ $? != 1 ] &&  exit 1
-
-# Check ufbint().
-./test_bort_4 ufbint 1
-[ $? != 1 ] &&  exit 1
-
-# Check ufbqcp().
-./test_bort_4 ufbqcp 1
-[ $? != 1 ] &&  exit 1
-
-# Check ufbrep().
-./test_bort_4 ufbrep 1
-[ $? != 1 ] &&  exit 1
-
-# Check ufbrms().
-./test_bort_4 ufbrms 1
-[ $? != 1 ] &&  exit 1
-
-# Check ufbseq().
-./test_bort_4 ufbseq 1
-[ $? != 1 ] &&  exit 1
-
-# Check ufdump().
-./test_bort_4 ufdump 1
-[ $? != 1 ] &&  exit 1
-
-# Check upftbv().
-./test_bort_4 upftbv 1
-[ $? != 1 ] &&  exit 1
-
-# Check valx().
-./test_bort_4 valx 1
-[ $? != 1 ] &&  exit 1
-
-# Check wrdxtb().
-./test_bort_4 wrdxtb 1
-[ $? != 1 ] &&  exit 1
-
-# Check wtstat().
-./test_bort_4 wtstat 1
-[ $? != 1 ] &&  exit 1
-./test_bort_4 wtstat 2
-[ $? != 1 ] &&  exit 1
-./test_bort_4 wtstat 3
-[ $? != 1 ] &&  exit 1
-./test_bort_4 wtstat 4
-[ $? != 1 ] &&  exit 1
-
-# Check writdx().
-./test_bort_4 writdx 1
-[ $? != 1 ] &&  exit 1
-
-# Check writlc().
-./test_bort_4 writlc 1
-[ $? != 1 ] &&  exit 1
-
-# Check writsa().
-./test_bort_4 writsa 1
-[ $? != 1 ] &&  exit 1
-
-# Check writsb().
-./test_bort_4 writsb 1
-[ $? != 1 ] &&  exit 1
+for kind in "4" "d" "8"; do
+    # Check adn30().
+    (./test_bort_$kind adn30 1) && exit 1
+    (./test_bort_$kind adn30 2) && exit 1
+    (./test_bort_$kind adn30 3) && exit 1
+    (./test_bort_$kind adn30 4) && exit 1
+    (./test_bort_$kind adn30 5) && exit 1
+
+    # Check bort().
+    (./test_bort_$kind bort 1) && exit 1
+
+    # Check bort2().
+    (./test_bort_$kind bort2 1) && exit 1
+
+    # Check bvers().
+    (./test_bort_$kind bvers 1) && exit 1
+
+    # Check cmpmsg().
+    (./test_bort_$kind cmpmsg 1) && exit 1
+
+    # Check codflg().
+    (./test_bort_$kind codflg 1) && exit 1
+
+    # Check copybf().
+    (./test_bort_$kind copybf 1) && exit 1
+
+    # Check copymg().
+    (./test_bort_$kind copymg 1) && exit 1
+
+    # Commented out until
+    # https://github.com/NOAA-EMC/NCEPLIBS-bufr/issues/395 can be
+    # resolved.
+    # Check copysb().
+    #./test_bort_$kind copysb 1
+    #[ $? != 1 ] &&  exit 1
+
+    # Check idn30().
+    (./test_bort_$kind idn30 1) && exit 1
+    (./test_bort_$kind idn30 2) && exit 1
+    (./test_bort_$kind idn30 3) && exit 1
+    (./test_bort_$kind idn30 4) && exit 1
+
+    # Check nemtba().
+    (./test_bort_$kind nemtba 1) && exit 1
+
+    # Check nemtbax(). Commented out, see
+    # https://github.com/NOAA-EMC/NCEPLIBS-bufr/issues/401.
+    #(./test_bort_$kind nemtbax 1) && exit 1
+    n
+    # Check nemtbb().
```

Comment:
Please correct me if I'm wrong, but since this is still commented out in the `run_test_bort.sh` script, then you're still not actually testing the nemtbax part of `test_bort.F90`, correct?  Don't you need to uncomment this too?

Reply:
Now removed.